File: problems/3516-find-closest-person/solution.c
int findClosest(int x, int y, int z) {
    int dist_x = abs(x - z);
    int dist_y = abs(y - z);

    if (dist_x < dist_y) {
        return 1;
    } else if (dist_y < dist_x) {
        return 2;
    } else {
        return 0;
    }
}

File: problems/3516-find-closest-person/solution.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        dist_x = abs(x - z)
        dist_y = abs(y - z)

        if dist_x < dist_y:
            return 1
        elif dist_y < dist_x:
            return 2
        else:
            return 0